pom.xml

```diff
@@ -198,12 +248,14 @@
         <module>cas-client-integration-jboss</module>
         <module>cas-client-support-distributed-ehcache</module>
         <module>cas-client-support-distributed-memcached</module>
+        <module>cas-client-support-saml</module>
```

Comment:
Same issue as with commons-lang: moving to saml produces compile errors. CommonUtils needs Joda.